api/swagger-spec/v1.json

```diff
@@ -12064,32 +12061,25 @@
       "description": "a description of the persistent volume's resources and capacityr; see http://releases.k8s.io/HEAD/docs/persistent-volumes.md#capacity"
      },
      "gcePersistentDisk": {
-      "$ref": "v1.GCEPersistentDiskVolumeSource",
-      "description": "GCE disk resource provisioned by an admin; see http://releases.k8s.io/HEAD/docs/volumes.md#gcepersistentdisk"
+      "$ref": "v1.GCEPersistentDiskVolumeSource"
      },
      "awsElasticBlockStore": {
-      "$ref": "v1.AWSElasticBlockStoreVolumeSource",
-      "description": "AWS disk resource provisioned by an admin; see http://releases.k8s.io/HEAD/docs/volumes.md#awselasticblockstore"
+      "$ref": "v1.AWSElasticBlockStoreVolumeSource"
      },
      "hostPath": {
-      "$ref": "v1.HostPathVolumeSource",
-      "description": "a HostPath provisioned by a developer or tester; for develment use only; see http://releases.k8s.io/HEAD/docs/volumes.md#hostpath"
+      "$ref": "v1.HostPathVolumeSource"
      },
      "glusterfs": {
-      "$ref": "v1.GlusterfsVolumeSource",
-      "description": "Glusterfs volume resource provisioned by an admin; see http://releases.k8s.io/HEAD/examples/glusterfs/README.md"
+      "$ref": "v1.GlusterfsVolumeSource"
      },
      "nfs": {
-      "$ref": "v1.NFSVolumeSource",
-      "description": "NFS volume resource provisioned by an admin; see http://releases.k8s.io/HEAD/docs/volumes.md#nfs"
+      "$ref": "v1.NFSVolumeSource"
      },
      "rbd": {
-      "$ref": "v1.RBDVolumeSource",
-      "description": "rados block volume that will be mounted on the host machine; see http://releases.k8s.io/HEAD/examples/rbd/README.md"
+      "$ref": "v1.RBDVolumeSource"
      },
      "iscsi": {
-      "$ref": "v1.ISCSIVolumeSource",
-      "description": "an iSCSI disk resource provisioned by an admin"
```

Comment:
Sorry for my late response. I also saw that but in the beginning I thought that those descriptions are dumped as they are describing a `$ref` (which `$ref` has already a description). But the more I dig in, this might be a bug. [Swagger spec](https://github.com/swagger-api/swagger-spec/blob/master/versions/1.2.md#529-property-object) doesn't say anything explicit about `$ref`s so I presume that description can we used.